mbs_results/constrains.py

```diff
@@ -0,0 +1,170 @@
+import operator
+from typing import List
+
+import pandas as pd
+
+
+def replace_values_index_based(
+    df: pd.DataFrame, target: str, a: int, compare: str, b: int
+) -> None:
+    """
+    Perform comparisons between a subset of df with a and subset of df with b and
```

Comment:
subset of df with "column" a? If not column can you add a little more context to a and b in this sentence?

Reply:
Yeah re-reading it now and doesn't make a lot of sense :D , will update it